proposals/2022/Add DiscountRate.md

```diff
@@ -0,0 +1,37 @@
+# Add Discount Rate
+
+## Overview
+
+This proposal is support the assertion of discount rates in BuildingSync XML schema.
+
+## Justification
+
+### Present Value Calculation
+
+The equation for net present value is:
+
+$$ NPV\left({i,N}\right) = \sum_{t=0}^{N}{\frac{R_{t}}{\left({1+i}\right)^t}} $$
+
+where
+* $i$ is the discount rate (percentage).
```

Comment:
Should we say i is the interest rate here since  v is the discount rate where v=1/(1+i)?